src/components/CustomMUIComponents/CustomButtons.js

```diff
@@ -28,6 +28,19 @@ export const PrimaryButton2 = ({content, width, onClick}) => {
         </Button>
     );
 }
+export const PrimaryButton3 = ({content, width, onClick}) => {
+    return (
+        <Button
+            onClick={onClick}
+            variant="contained"
+            style={{
+                paddingTop: '10px', paddingBottom: '10px',
```

Comment:
for consistency, put one attribute per line

Reply:
in my IDE i see all the formatting the same was 
<img width="566" alt="Screen Shot 2022-11-02 at 7 42 46 PM" src="https://user-images.githubusercontent.com/43822671/199622623-43d7a06c-d75b-4906-aacb-43ddfd7bea51.png">

Reply:
I see all the buttons that way in my IDE
<img width="541" alt="Screen Shot 2022-11-02 at 7 44 29 PM" src="https://user-images.githubusercontent.com/43822671/199622798-240154d0-b389-493e-91ae-d2ac67c1aa1c.png">

<img width="517" alt="Screen Shot 2022-11-02 at 7 45 43 PM" src="https://user-images.githubusercontent.com/43822671/199622901-4aa2360e-c078-42f8-a7ef-375b3d297b7a.png">